### DISCUSSION TITLE
raspotify conf file

### DISCUSSION THREAD

**Discussion by pavkaolsan** [dtcooper/raspotify#497] [Q&A] [answered]
Hello, I am new to the Raspberry PI OS. I installed the raspotify. Streaming over the LAN is working fine. I would like to add username and password to play it with Spotify Connet, but I cannot access the folder /etc/raspotify/ (that folder seems to be empty). Have I done something wrong over the installation? Thanks in advance

**Comment by JasonLG1979** [comment, marked as answer]
> Streaming over the LAN is working fine. I would like to add username and password to play it with Spotify Connet,

You really don't need to add your credentials unless you want to disable zeroconf discovery. If you're streaming over LAN you're already connected via Spotify Connect.

> but I cannot access the folder /etc/raspotify/ (that folder seems to be empty).

The `/etc/raspotify` folder is owned by root with `600` permissions meaning that only root can read and write to the folder. The reason being that there may be credentials in the `/etc/raspotify/conf` file.

You will need root privileges to read/write `/etc/raspotify/conf`.

Raspberry Pi OS comes with [nano](https://linux.die.net/man/1/nano).

You can use nano to edit `/etc/raspotify/conf`:

```
sudo nano /etc/raspotify/conf
```

**Reply by pavkaolsan** [reply]
Yeah, I tried the command: sudo nano etc/raspotify/conf But that opens only empty file. I would like to increase the bitrate. If possible

**Reply by pavkaolsan** [reply]
My bad, now I can open that file. Thanks for help
